src/gui/ControllerRackView.cpp

```diff
@@ -141,9 +169,26 @@ void ControllerRackView::onControllerAdded( Controller * controller )
 
 	auto controllerView = new ControllerView(controller, scrollAreaWidget);
 
+	connect(controllerView, SIGNAL(moveUp(lmms::gui::ControllerView* )), this, 
+		SLOT(moveUp(lmms::gui::ControllerView* )), Qt::QueuedConnection);
+	connect(controllerView, SIGNAL(moveDown(lmms::gui::ControllerView* )), this, 
+		SLOT(moveDown(lmms::gui::ControllerView* )), Qt::QueuedConnection);
 	connect( controllerView, SIGNAL(deleteController(lmms::gui::ControllerView*)),
 		 this, SLOT(deleteController(lmms::gui::ControllerView*)), Qt::QueuedConnection );
 
+	QAction* moveUpAction = new QAction(controllerView);
+	moveUpAction->setShortcut(Qt::Key_Up | Qt::AltModifier);
+	moveUpAction->setShortcutContext(Qt::WidgetShortcut);
+	connect(moveUpAction, SIGNAL(triggered()), controllerView, SLOT(moveUp()));
+	controllerView->addAction( moveUpAction );
```

Comment:
Unnecessary white space in new code. :wink:

Reply:
removed the extra space.